eng/targetingpacks.targets

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup Condition="'$(DisableImplicitFrameworkReferences)' != 'true' and
                             '$(TargetFrameworkIdentifier)' == '.NETCoreApp' and
                             $([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), '$(NETCoreAppCurrentVersion)'))">
-    <_UseLocalTargetingRuntimePack>true</_UseLocalTargetingRuntimePack>
+    <UseLocalTargetingRuntimePack Condition="'$(UseLocalTargetingRuntimePack)' == ''">true</UseLocalTargetingRuntimePack>
     <EnableTargetingPackDownload>false</EnableTargetingPackDownload>
```

Comment:
I think we still want to download the targeting pack if UseLocalTargetingRuntimePack=false.  It's possible that the SDK doesn't carry it (once it moves to `net6.0` for example).